src/components/tooltip/README.md

```diff
@@ -111,3 +111,4 @@ In the following example, the tooltip text only appears on hover when the button
 | `horizontalConstraint` | `object` |    -     | `xs`, `s`, `m`, `l`, `xl`, `scale`                                                                                                           | `scale` | Horizontal size limit of the tooltip                                                            |
 | `children`             | `node`   |    ✅    | -                                                                                                                                            | -       | Content rendered within the tooltip                                                             |
 | `components`           | `object` |    -     | `WrapperComponent`, `BodyComponent`                                                                                                          | -       | If passed, the tooltip will wrap your component with this element                               |
+| `customBodyStyles`     | `object` |    -     | -                                                                                                                                            | -       | If passed, these styles will be spread onto the div surrounding the tooltip body                |
```

Comment:
Not sure if I am overly picky with the name again. But given we tend to like the `components={{header, footer}}` pattern. How about going with `styles={ body }` here?